source/_animate_info.html.erb

```diff
@@ -0,0 +1,3 @@
+<div class="animate-info">
+  This component is completely based on Dan Eden's work Animate.css. It sets up the JavaScript for triggering an animation for another element. Go to <a href="https://github.com/daneden/animate.css/blob/master/animate.css">the project GitHub page</a> for other animation types. Or tweak the one above. For more animation examples, go to <a href="http://daneden.github.io/animate.css/">Animate.css</a>
```

Comment:
- No need for “completely”
- Dan Eden's » Dan Eden’s (proper smart quote; Dan is a type-nerd, so he’ll appreciate it :smile:)
- No need for “work”
- I think the last two sentences could be condensed to:

> Check out the <a href="http://daneden.github.io/animate.css">Animate.css</a> website to view the animations in action and the <a href="https://github.com/daneden/animate.css/blob/master/animate.css">GitHub page</a> for the CSS.